apps/docs/src/common/pages/patterns/access-a-service.tsx

```diff
@@ -0,0 +1,119 @@
+        <img src={applyAuthenticatePersonal} className="image-examples" alt="An example of a service verifying the user with passport details and date of birth" />
+      </div>
+
+      <h2 className="govuk-heading-l">Access a service with security codes</h2>
+      <p>You can use security codes or two-factor authentication (often shortened to 2FA) to verify a user's identity to let them access a service.</p>
+      <p>You can use this method of authentication when a service stores user data and allows users to access it. Two-factor helps protect the user and the service.</p>
+      <p>There are three types of two-factor authentication:</p>
+      <ul className="govuk-list govuk-list--bullet">
+        <li>something a user knows (password or pin)</li>
+        <li>something a user has (phone or fob)</li>
+        <li>a biometric (fingerprint or voice)</li>
+      </ul>
+      <div className="example example-images confirmation" style={{marginBottom: "2em"}}>
+        <img src={applyAuthenticateTwoFa} className="image-examples" alt="An example of a service verifying the user with two-factor authentication" />
+      </div>
+
+      <h3 className="govuk-heading-m">Services using these patterns</h3>
+      <ul className="govuk-list govuk-list--bullet">
+        <li>Access UK</li>
+        <li>Apply for the EU settlement scheme</li>
+        <li>Employer checking service</li>
+      </ul>
+
+      <h2 className="govuk-heading-l">Access a service with reference numbers</h2>
+      <p>A reference number is a simple way to let users access a service. For example, when a user saves progress and needs to return to complete a service.</p>
+      <div className="example example-images confirmation" style={{marginBottom: "2em"}}>
+        <img src={applyAuthenticateReference} className="image-examples" alt="An example of a service verifying the user with reference number" />
+      </div>
+
+      <h3 className="govuk-heading-m">Services using this pattern</h3>
+      <ul className="govuk-list govuk-list--bullet">
+        <li>Passport renewals</li>
+      </ul>
+
+      <h2 className="govuk-heading-l">Research</h2>
+      <p>More research is needed. If your service uses this pattern, get in touch to share your user research findings.</p>
+
+      <h2 className="govuk-heading-l">Accessibility</h2>
+      <p>If your service uses this pattern, let us know of any insights you have on accessibility considerations.</p>
+
+      <div className="contact-us">
+        <h2 className="govuk-heading-m">Get in touch</h2>
+        <p>If you’ve got a question or suggestion share it on the Slack channel #ho-design-system, on <A href="https://github.com/UKHomeOffice/design-system">GitHub</A> or email <A href="mailto:design@digital.homeoffice.gov.uk">design@digital.homeoffice.gov.uk</A>.</p>
+      </div>
+
+    </div>
+</div>
+
+
+);
+
+export default Page;
+export const title = 'Images';
```

Comment:
Access a service?

Reply:
Yes. Have changed this across all the patterns accordingly